products/compute/ansible.yaml

```diff
@@ -130,6 +130,8 @@ overrides: !ruby/object:Provider::ResourceOverrides
     properties:
       target: !ruby/object:Provider::Ansible::PropertyOverride
         version_added: '2.7'
+      allPorts: !ruby/object:Provider::Ansible::PropertyOverride
```

Comment:
Is this necessary since it won't actually get generated in ansible?

Reply:
This was added to address a breakage in the Ansible CI pipeline for this PR. Alex helped me with this, I'll loop him in so he can provide the expert explanation for it. @rambleraptor

Reply:
@craigatgoogle can you try removing that line and re-running the Magician? I agree with Dana. The lack of an Ansible generated PR means that this isn't changing anything on Ansible, so changes to `products/compute/ansible.yaml` shouldn't be necessary.

Reply:
Thanks Dana and Alex. I've removed the change to the ansible config. I'll circle back around to this if the CI issues manifest again.